docs/source/API/types/types.rst

```diff
@@ -2,3 +2,50 @@
 
 Types
 =====
+
+.. contents:: 
+    :local: 
+
+Starfish uses a series of constants and container types to enable type checking, and provides a set
+of useful hints about how to interact with starfish objects. 
+
+Coordinates
+-----------
+
+Coordinates holds constants that store with the physical coordinates of a field of view. They define
+a field of view's relative location to some global scale parameter, and identify how to stitch or 
+combine multiple fields of view. 
+
+.. literalinclude:: /../../starfish/types/_constants.py
+    :pyobject: Coordinates
+
+Indices
+-------
+
+Indices holds constants that represent indexers into the dimensions of the ImageStack 5-d image 
+tensor. They are re-used by objects that inherit subsets of these Indices, such as the 
+IntensityTable, which stores spot coordinates and pixel traces across rounds and channels,  
```

Comment:
pixel traces?

Reply:
See glossary section: d82aa01

When the docs are all pushed, we can add hyperlink cross-references. :-)